### PR TITLE
[dashboard] add regular workspaces to list only

### LINKED ISSUE #4436: [dashboard] Headless workspaces shouldn't be displayed in the workspaces list
### Bug description

https://gitpod.io/workspaces should only display regular workspaces.

However, if you start a prebuild (or if an app/webhook you've installed triggers a prebuild for you), you'll see it running in your list of workspaces.

### Steps to reproduce

1. `https://gitpod.io/#prebuild/<some-repo>`
2.

Screenshot 2021-06-08 at 14 10 11

### Expected behavior

The prebuild in progress should not be visible in workspaces list.

(Instead, it should be listed in the upcoming prebuilds dashboard.)

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Fixes #4436

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5001#" title="Author self-approved">JanKoehnlein</a>*

Associated issue: *#4436*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/workspaces/workspace-model.ts [hunk 1]
```diff
@@ -69,8 +69,10 @@ export class WorkspaceModel implements Disposable, Partial<GitpodClient> {
                 try {
                     this.currentlyFetching.add(instance.workspaceId);
                     const info = await getGitpodService().server.getWorkspace(instance.workspaceId);
-                    this.workspaces.set(instance.workspaceId, info);
-                    this.notifyWorkpaces();
+                    if (info.workspace.type == 'regular') {
```
Comment (csweichel) [inline components/dashboard/src/workspaces/workspace-model.ts:72]: nit
```suggestion
                    if (info.workspace.type === 'regular') {
```

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 4c8278920252704a0cd9be1bcc86bbab329e11b1</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5001#issuecomment-890571460" title="LGTM">csweichel</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5001#" title="Author self-approved">JanKoehnlein</a>*

Associated issue: *#4436*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein,csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**